osdep/io.h

```diff
@@ -130,7 +130,9 @@ char *mp_getenv(const char *name);
 #define environ (*mp_penviron())  /* ensure initialization and l-value */
 char ***mp_penviron(void);
 
-off_t mp_lseek(int fd, off_t offset, int whence);
+#undef off_t
+#define off_t int64_t
```

Comment:
make it conditional to the affected platforms?

Reply:
It is already in _WIN32 ifdef and we don't support building without `_FILE_OFFSET_BITS=64` anyway. mp_lseek is unconditionally calling _lseeki64